4/4_3/exo_Part_1_5.js

```diff
@@ -0,0 +1,27 @@
+//Excercice 5
+console.log("Exercicio 5");
+var info = {
+    personagem: "Margarida",
+    origem: "Pato Donald",
+    nota: "Namorada do personagem principal nos quadrinhos do Pato Donald",
+    recorrente: 'Sim'
+  };
+  
+  var info2 = {
+    personagem: "Tio Patinhas",
+    origem: "Christmas on Bear Mountain, Dell’s Four Color Comics #178",
+    nota: "O último MacPatinhas",
+    recorrente: 'Sim'
+  };
+
+console.log("resultado dos valores de info usando For In:");
+console.log("");
```

Comment:
console.log("resultado dos valores de info usando For In:\n");

Reply:
o back slash "\" seguido de "n" pula uma linha ;)
Acho mais interessante colocar o \n no final do console.log da linha 17 e retirar o console.log da linha 18